assets/dev/scss/editor/panel/controls/_media.scss

```diff
@@ -201,7 +201,7 @@
 			}
 
 			&__remove {
-				@include border-end(var(--e-a-border));
```

Comment:
can you remove all of those `(padding | margin | border)-(start | end)` functions definitions?

Reply:
No, not yet. We are still using them in front-end widgets. This PR fixes only the Elementor Editor.